client/errs/errs.go

```diff
@@ -38,6 +38,15 @@ func IsLeaderChange(err error) bool {
 		strings.Contains(errMsg, NotPrimaryErr)
 }
 
+func IsServiceModeChange(err error) bool {
+	if err == nil {
+		return false
+	}
+	errMsg := err.Error()
+	return strings.Contains(errMsg, "not found tso address") ||
+		strings.Contains(errMsg, "maximum number of retries exceeded")
```

Comment:
This error would also occur when the leadership cannot be elected. In which case will this be a misjudgment?

Reply:
We don't check this error on client side, do you know about the reason?